[Install single-server OpenStack with conjure-up] Proxy settings

While trying to install OpenStack as it is documented in this tutorial, I ran into the problem that juju bootstrapping did not work. I got the following error:

```
2018-02-13 13:49:58,458 [DEBUG] conjure-up/openstack-novalxd - juju.py:187 - Bootstrapping to set region: {}
2018-02-13 13:49:58,458 [DEBUG] conjure-up/openstack-novalxd - juju.py:223 - bootstrap cmd: ['/snap/bin/juju', 'bootstrap', 'localhost/localhost', 'conjure-up-localhost-1c2', '--default-model', 'conjure-openstack-novalx-2c0', '--config', 'image-stream=daily', '-- PROXY SETTINGS (I deleted the IPs)', '--bootstrap-series', 'xenial']
2018-02-13 13:49:58,473 [DEBUG] conjure-up/openstack-novalxd - events.py:53 - Awaiting Bootstrapped at conjureup/controllers/bootstrap/tui.py:17
2018-02-13 13:50:12,615 [ERROR] conjure-up/openstack-novalxd - common.py:52 - Error bootstrapping controller: ['Creating Juju controller "conjure-up-localhost-1c2" on localhost/localhost', 'Looking for packaged Juju agent version 2.3.1 for amd64', 'To configure your system to better support LXD containers, please see: https://github.com/lxc/lxd/blob/master/doc/production-setup.md', 'Launching controller instance(s) on localhost/localhost...', '   ', ' - unable to get LXD image for juju/xenial/amd64: Get https://cloud-images.ubuntu.com/daily//streams/v1/index.json: Unable to connect to: cloud-images.ubuntu.com:443', 'ERROR failed to bootstrap model: cannot start bootstrap instance: unable to get LXD image for juju/xenial/amd64: Get https://cloud-images.ubuntu.com/daily//streams/v1/index.json: Unable to connect to: cloud-images.ubuntu.com:443']
2018-02-13 13:50:12,617 [DEBUG] conjure-up/openstack-novalxd - events.py:53 - Setting Error at conjureup/events.py:148
2018-02-13 13:50:12,618 [ERROR] conjure-up/openstack-novalxd - events.py:160 - Unhandled exception in <Task finished coro=<BaseBootstrapController.run() done, defined at /snap/conjure-up/924/lib/python3.6/site-packages/conjureup/controllers/bootstrap/common.py:15> exception=BootstrapError('Unable to bootstrap (cloud type: localhost)',)>
Traceback (most recent call last):
  File "/snap/conjure-up/924/lib/python3.6/site-packages/conjureup/controllers/bootstrap/common.py", line 21, in run
    await self.do_bootstrap()
  File "/snap/conjure-up/924/lib/python3.6/site-packages/conjureup/controllers/bootstrap/common.py", line 57, in do_bootstrap
    app.provider.cloud_type))
conjureup.errors.BootstrapError: Unable to bootstrap (cloud type: localhost)
```

After some searching in the juju bug tracker I found [this bug](https://bugs.launchpad.net/juju/+bug/1594720) where a comment described how to set the proxy settings for lxd:

```
lxc config set core.proxy_http=
lxc config set core.proxy_https=
lxc config set core.proxy_ignore_hosts=
```

After setting these variables I don't get any errors like this. Sadly these settings are not documented well for lxd, so it would be great if this was mentioned in the tutorial.

--- a/tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md
+++ b/tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md
@@ -83,7 +83,7 @@ sudo usermod -a -G lxd <username>
 newgrp lxd
 ```
 
-If you are using a proxy server, you might run into errors if you do not configure lxd specifically. To do so use the following commands:
+If you are using a proxy server, you might run into errors if you do not configure LXD specifically. To do so use the following commands:
 
 ```bash
 lxc config set core.proxy_http <http proxy server>

--- a/tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md
+++ b/tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md
@@ -83,6 +83,14 @@ sudo usermod -a -G lxd <username>
 newgrp lxd
 ```
 
+If you are using a proxy server, you might run into errors if you do not configure lxd specifically. To do so use the following commands:
+
+```bash
+lxc config set core.proxy_http <http proxy server>
+lxc config set core.proxy_https <https proxy server>
+lxc config set core.proxy_ignore_hosts <no_proxy hosts>
+```
+
 ## Installing OpenStack
 Duration: 45:00
 

